ios/RNSScreen.m

```diff
@@ -402,6 +409,23 @@ - (BOOL)prefersStatusBarHidden
   return screenView.statusBarHidden;
 }
 
+- (UIInterfaceOrientationMask)supportedInterfaceOrientations
+{
+  UIViewController *presentedVC = [self presentedViewController];
+  if ([presentedVC isKindOfClass:[RNSScreen class]]) {
+    return presentedVC.supportedInterfaceOrientations;
+  }
+  
+  UIViewController *child = [[self childViewControllers] lastObject];
+  if ([child isKindOfClass:[RNScreensNavigationController class]] || [child isKindOfClass:[RNScreensViewController class]]) {
```

Comment:
What is `RNScreensViewController` and `RNScreensNavigationController` ?

https://github.com/software-mansion/react-native-screens/search?q=RNScreensNavigationController

Reply:
They are subclasses for `ScreenContainer`'s `UIViewController` and `ScreenStack`'s `UINaviagationController` added in this PR to override methods connected to orientation and status bar managment.